exercises/clock/clock_test.go

```diff
@@ -51,6 +51,13 @@ func TestAddMinutes(t *testing.T) {
 				a.h, a.m, a.a, got, a.want)
 		}
 	}
+	// TODO: Don't duplicate.
```

Comment:
OK sure, if you are OK with it, I will simply remove the comment and we will be done with it. It will be fine to have this for now. I was dissatisfied with the fact that lines 56 - 59 are the same as lines 49-52.